python/cudf/cudf/core/dataframe.py

```diff
@@ -1700,9 +1697,31 @@ def _concat(cls, objs, axis=0, ignore_index=False):
             )
         else:
             index = Index._concat([o.index for o in objs])
+
+        # Currently we only support sort = False
+        # Change below when we want to support sort = True
+        # below functions as an ordered set
+        all_columns_ls = [col for o in objs for col in o.columns]
+        unique_columns_ordered_ls = OrderedDict.fromkeys(
+            all_columns_ls, None
```

Comment:
The None is optional here, so you can remove it:
```suggestion
            all_columns_ls
```

Reply:
@VibhuJawa just waiting on the removal of this for merge.

Reply:
Should be fixed with https://github.com/rapidsai/cudf/commit/5b068133e6800e65603c2265876870580a2f35a0 .